feature-examples/python/corpus/compute_corpus_size.py

```diff
@@ -0,0 +1,69 @@
+"""Example of using the Vectara REST API to compute corpus size."""
+
+import dataclasses
+import logging
+
+import requests
+
+from corpus import exceptions
+
+@dataclasses.dataclass(frozen=True)
+class CorpusSize:
+    """Corpus Size and timestamp when it was computed."""
+
+    epoch_secs: int
+    size: int
+
+
+def compute_corpus_size(
+    customer_id: int,
+    corpus_id: int,
+    jwt_token: str,
+) -> CorpusSize:
+    """Computes a corpus size.
+
+    Args:
+        customer_id: Unique customer ID in vectara platform.
+        corpus_id: Corpus ID for which size needs to be computed.
+        jwt_token: JWT token to be used for authentication.
+
+    Returns:
+        CorpusSize object
+
+    Raises:
+        CorpusException: In case of any error.
+    """
+    post_headers = {
+        "customer-id": f"{customer_id}",
+        "Authorization": f"Bearer {jwt_token}",
+    }
+
+    request = { "corpusId": corpus_id }
```

Comment:
no spaces after `{` or before `}`

https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements